recipes-ni/ni-configpersistentlogs/ni-configpersistentlogs_1.0.bb

```diff
@@ -0,0 +1,28 @@
+SUMMARY = "NI persistent logs configuration utility"
```

Comment:
Sorry; I should have noted this before. Since all of the source for `ni-configpersistentlogs` lives in OE directly, encoding the version name into the recipe filename doesn't do much. It just means that, if a future dev does want to rev the package version, they'll have to do a git-rename operation, which can complicate the git history.

Instead, you can just rename the recipe file to `ni-configpersistentlogs.bb` and add a `PV = "1.0"` line in the recipe. Then devs can just update that variable.